edgelet/contrib/centos/aziot-edge.spec

```diff
@@ -10,14 +10,14 @@ Name:           aziot-edge
 Version:        @version@
 Release:        @release@%{?dist}
 
-License:        Proprietary
+License:        MIT
 Summary:        Azure IoT Edge Module Runtime
 URL:            https://github.com/azure/iotedge
 
 %{?systemd_requires}
 BuildRequires:  systemd
 Requires(pre):  shadow-utils
-Requires:       aziot-identity-service >= @version@-@release@
+Requires:       aziot-identity-service >= 1.2.0~rc4-1
```

Comment:
Make it `=` just in case there's some version shenanigans with the final release (`1.2.0~rc4-1` sorts higher than `1.2.0-1` or something)

Reply:
Sure, although I don't plan to merge this PR. Just made it so I could run the pipeline to generate test packages.